src/app/services/media-to-grid-media.service.ts

```diff
@@ -5,7 +5,7 @@ import { Movie } from '../types/movie';
 import { Person } from '../types/person';
 import { TvShow } from '../types/tv-show';
 import { Cast } from '../types/credits';
-import { isCast } from 'app/guards/cast-type-guard';
+import { isCast } from 'src/app/guards/cast-type-guard';
```

Comment:
It's a bit odd

Reply:
Don't know how it happend, was shocked aswell.